src/ArrayUtils.f90

```diff
@@ -991,4 +995,69 @@ PURE FUNCTION findUpBound_1DReal(r,pos,delta,incl,tol) RESULT(val)
   ENDIF
 ENDFUNCTION findUpBound_1DReal
 !
+!-------------------------------------------------------------------------------
+!> @brief Routine that checks that the entries of r either monotonically increase or decrease.
+!>        Note this is limited to 1 dimensional real arrays.
+!> @param r input array to check for monotonicity of values
+!> @returns good boolean indicating whether the check has passed or not
```

Comment:
don't personally love using the name "good"...@ag6...thoughts?

Reply:
Personally I don't really care.  For a routine that's checkign things, "good" seems to be clear enough to me.  But I'm open to a change as well.